src/js/options.ts

```diff
@@ -121,17 +121,29 @@ export interface BaseReactNativeOptions {
    * @default 2
    */
   appHangTimeoutInterval?: number;
+
+  /**
+   * The max queue size for capping the number of envelopes waiting to be sent by Transport.
+   */
+  maxQueueSize?: number;
+}
+
+export interface ReactNativeTransportOptions extends BrowserTransportOptions {
+  /**
+   * @deprecated use `maxQueueSize` in the root of the SDK options.
+  */
+  bufferSize?: number;
```

Comment:
By doing this, if I do:
```
Sentry.init({
	transport(transportOptions) {
	    transportOptions.bufferSize = 100;
	    // ...
	  },
}
```

Do I see in the auto-complete that `bufferSize` is deprecated? 
Should the linter work?
it's not working for me at least.

Reply:
I think it is some ts and lint caching problem, it works for me after reloading my VSC window.

<img width="488" alt="Screenshot 2022-11-04 at 14 28 05" src="https://user-images.githubusercontent.com/31292499/199984127-b6895938-f1ad-4a1f-8624-74a53e0b2432.png">

Try that I hope it will help.

Reply:
No need to block this, just wanted to be sure that it works.